generator/generator-core/src/main/resources/templates/config/modlPluginConfig.kts

```diff
@@ -25,7 +25,7 @@ ignitionModule {
      * the course of a major Ignition (7.9, 8.0, etc) version, except for when the Ignition Platform adds/changes APIs
      * used by the module.
      */
-    requiredIgnitionVersion.set(rootProject.ext.sdk_version)
+    requiredIgnitionVersion.set(sdk_version)
```

Comment:
I should never have patched this template in #40. There was only a problem with the `modlPluginConfig.groovy` template, not this Kotlin DSL template.